mlir/test/Dialect/MIOpen/lowering_bound_check.mlr

```diff
@@ -0,0 +1,22 @@
+//RUN:mlir-opt ./mytest.mlir %s -miopen-affine-transform |FileCheck %s --check-prefix=CHECK_AFFINE
```

Comment:
`./mytest.mlir` is incorrect. should be `%s`.

Reply:
done